src/AppBundle/Controller/BuilderController.php

```diff
@@ -227,6 +227,40 @@ public function octgnexportAction($deck_id)
         return $response;
     }
 
+    public function textorderedexportAction($deck_id, $deluxeAfter)
```

Comment:
needs equivalent action in `SocialController` in order to allow this kind of download for published decklists. Note that the new controller action will take a decklist id, rather than the deck id, which adds another level of indirection.

Reply:
Solved in 6e6c54df53086d6174d428bc51feab923d66d56f